packages/core/src/components/StepList/StepList.example.jsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/display-name */
+import React from 'react';
+import StepList from './StepList';
+
+export default function() {
+  return (
+    <div style={{ maxWidth: '628px' }}>
+      <StepList
+        inverse
+        completedText="Completed"
+        editText="Edit"
+        resumeText="Resume"
+        startText="Start"
+        onEnterStep={() => {}}
```

Comment:
This could be potentially confusing to folks. I'd rather have this do something, or not show it if it's an empty function.